src/mediaplayer.cs

```diff
@@ -235,20 +240,28 @@ public interface MPMediaItem {
 		NSString HasProtectedAssetProperty { get; }
 	}
 
-	[NoTV]
 	[BaseType (typeof (NSObject))]
 	[DisableDefaultCtor]
 	public interface MPMediaItemArtwork {
+		[iOS (10,0)]
+		[Export ("initWithBoundsSize:requestHandler:")]
 		[DesignatedInitializer]
+		IntPtr Constructor (CGSize boundsSize, Func<CGSize, UIImage> requestHandler);
+
+		[Introduced (PlatformName.iOS, 5, 0)]
```

Comment:
you don't have to add stuff before 6.0 (the oldest iOS version we support) as it can't really be used to discriminate which API should be used when developing